.github/workflows/go.yml

```diff
@@ -0,0 +1,22 @@
+name: Go
+
+on:
+  push:
+    branches: [ master, v3 ]
+  pull_request:
+    branches: [ master, v3 ]
+
+jobs:
+
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: 1.15
```

Comment:
We should probably test against 1.16 now too.

Reply:
Do you want to do that in addition to `1.15` or should I just bump the verison?

Reply:
In additon. Let's test with the two most recent versions, as that's what to Go team supports upstream.